src/styles/block/catalog--card-stule.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  box-sizing: border-box;
+  height: 408px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 32px 0 36px;
+    text-align: center;
+  }
+
+  &__photo {
+    height: 134px;
+  }
+
+  &__info {
+    margin-inline: 16px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: $second-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__grade {
+    line-height: 16px;
+  }
+
+  &__prices {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &__text {
+    font-size: 12px;
+    color: $second-text-color;
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 40px;
+    text-align: center;
+    background-color: $button-main-color;
+    border-radius: 5px;
+    cursor: pointer;
+  }
+
+  &__buy {
+    text-decoration: none;
+    text-transform: uppercase;
+    color: $button-second-color;
+  }
+
+  &__button:hover {
+    background-color: $button-second-color;
+    border: 1px solid $button-main-color;
+    transition-duration: 1s;
```

Comment:
The `transition-duration` should be defined within the original element state, not the hovered state. This ensures the transition is smooth in both directions, from normal to hover and hover to normal.

Reply:
apply it

![image](https://github.com/user-attachments/assets/bdbbe152-d342-4882-a161-2c4dbbee6812)